packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -93,15 +92,16 @@ import { getLocalizedString } from "../../common/localizeUtils";
 const Constants = {
   vscodeProvisionBicepPath: "./templates/azure/provision.bicep",
   launchJsonPath: ".vscode/launch.json",
-  appYmlName: "app.yml",
-  appLocalYmlName: "app.local.yml",
+  appYmlName: "teamsapp.yml",
+  appLocalYmlName: "teamsapp.local.yml",
```

Comment:
There is a file versionMetadata. It contains the above two constant. Just use them

Reply:
replaced